internal/pkg/template/input/input.go

```diff
@@ -82,11 +82,21 @@ type Input struct {
 	Rules       Rules       `json:"rules,omitempty" validate:"omitempty,template-input-rules"`
 	If          If          `json:"showIf,omitempty" validate:"omitempty,template-input-if"`
 	Options     Options     `json:"options,omitempty" validate:"template-input-options"`
-	ComponentId string      `json:"componentId,omitempty"`
+	ComponentId string      `json:"componentId,omitempty" validate:"required_if=Kind oauth"`
```

Comment:
Tohle znamená že `componentId` je povinný pokud `kind` je `oauth`. Je na to test dole v `internal/pkg/template/input/validation_test.go`